pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -34,6 +34,7 @@
 
   # Feature flags
 , withAlsa ? withHeadlessDeps && stdenv.isLinux # Alsa in/output supporT
+, withAmf ? withFullDeps # AMD Media Framework video encoding
```

Comment:
We could guard this behind `amf.meta.availableOn` as this only really makes sense for platforms for which drivers exist but there might be scenarios where the user has their own driver?

Reply:
I will switch it over for now and if someone has their own driver they can create an issue so that we can enable it for other platform.